testutil/keeper/unit_test_helpers.go

```diff
@@ -266,15 +271,6 @@ func TestProviderStateIsCleanedAfterConsumerChainIsStopped(t *testing.T, ctx sdk
 	acks := providerKeeper.GetSlashAcks(ctx, consumerId)
 	require.Empty(t, acks)
 
-	// in case the chain was successfully stopped, it should not contain a Top N associated to it
-	ps, err := providerKeeper.GetConsumerPowerShapingParameters(ctx, consumerId)
```

Comment:
We do not delete the power-shaping or initialization parameters during a chain deletion anymore.